Reply/app/src/main/java/com/example/reply/ui/EmptyComingSoon.kt

```diff
@@ -0,0 +1,41 @@
+package com.example.reply.ui.utils
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.example.reply.R
+
+@Composable
+@Preview
+fun EmptyComingSoon() {
```

Comment:
There are no screen for rest of the tabs in Figma sample except Home, I have asked designer if they will implement that or it meant to only show home screen with adaptive features.

Reply:
I think it's fine for now, we've done similar in other Compose samples eg. Jetchat